.github/codecov.yml

```diff
@@ -26,4 +26,7 @@ coverage:
         target: 100%
         threshold: 0%
         if_ci_failed: error
-        if_not_found: failure
\ No newline at end of file
+        if_not_found: failure
+    ignore:
+      - "*/tests/*"
+      - "*/tests/Test_"
```

Comment:
Why is this here? Test code will auto-execute and automatically count to coverage. It will also help flag if any test is not running for any reason, or if any code path in a test is dead-code.

If anything, things to be excluded from coverage should be non-code sections like the tools folder.